modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/objectViewContext.tsx

```diff
@@ -253,9 +265,10 @@ const viewReducer = (state: TState, action: TAction) => {
 						if (objectField.name === sortColumn.objectFieldName) {
 							newObjectViewSortColumns.push({
 								...sortColumn,
-								fieldLabel: objectField.label[
-									defaultLanguageId
-								] as string,
+								fieldLabel: getLocalizableLabel(
+									objectField.label,
+									creationLanguageId
+								),
```

Comment:
Hey @alinedoleron, you can use the `objectField.name` as the `fallback` of the function